src/generators.ts

```diff
@@ -100,12 +143,45 @@ export function createNotebookGenerator(
                 }
               };
             };
+            let lastLevel = Private.getLastLevel(headings);
+            headings = headings.concat(
+              Private.getMarkdownHeadings(
+                model.value.text,
+                onClickFactory,
+                numberingDict,
+                lastLevel
+              )
+            );
+          }
+        } else if (model.type === 'raw') {
+          let showRaw = false;
+          if (widget) {
+            showRaw = widget.showRaw;
+          }
+          if (showRaw) {
+            let text = (model as CodeCellModel).value.text;
+            const onClickFactory2 = (line: number) => {
```

Comment:
Can we rename this to `onClickFactory`, since there is not another one in scope?